### ISSUE TITLE
new feature about print frequency

### ISSUE THREAD

**Issue by CiscoRachel** [fmtlib/fmt#3612]
hello, there is exist way to control print frequency,like glog fucntion:LOG_EVERY_N(INFO,100)<<"xxxxx",print "xxxx" when counter is 100.

**Comment by vitaut** [comment]
No, but this could be a part of some logging library based on {fmt}.
